src/test/java/org/example/validators/EmailValidatorTest.java

```diff
@@ -0,0 +1,22 @@
+package org.example.validators;
+
+import org.junit.jupiter.api.Test;
+
+import static org.example.validators.EmailValidator.isValidEmail;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class EmailValidatorTest {
+
+    @Test
+    public void isValidEmail_shouldReturnTrue_forValidEmail() {
+        String VALID_EMAIL = "valid.address@gmail.com";
+        assertTrue(isValidEmail(VALID_EMAIL));
+    }
+
+    @Test
+    public void isValidEmail_shouldReturnFalse_forInvalidEmail() {
```

Comment:
Can you please add some more invalid tests as there could be a number of different things that make an email invalid

Reply:
great job